app/controller/cheets.rb

```diff
@@ -0,0 +1,8 @@
+class Chitter < Sinatra::Base
+
+  post('/cheets') do
+    text = params["text"]
+    Cheet.create(:text=> text, :user_id => session[:user_id], :time => Time.now)
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.